samples/sensor/thermometer/sample.yaml

```diff
@@ -41,3 +41,17 @@ tests:
       - platform:mimxrt1180_evk/mimxrt1189/cm33:SHIELD=p3t1755dp_ard_i3c
       - platform:mimxrt1180_evk/mimxrt1189/cm7:SHIELD=p3t1755dp_ard_i3c
       - platform:frdm_mcxn236/mcxn236:SHIELD=p3t1755dp_ard_i3c
+  sample.sensor.thermometer_i2c:
+    tags: sensors
+    harness_config:
+      fixture: sensor_ambient_temp_i2c
+      type: multi_line
+      regex:
+        - "Thermometer Example (.*)"
+        - "Temperature device is 0x[0-9|a-z]+, name is [a-z|0-9]+@[a-z|0-9]+"
+        - "Temperature is [0-9|.]+°C"
+    integration_platforms:
+      - frdm_rw612 # p3t1755
+    depends_on:
+      - i2c
+    filter: dt_alias_exists("ambient-temp0")
```

Comment:
This new configuration isn't needed, you can add `frdm_rw612` to the list of integration platforms in the `sample.sensor.thermometer` configuration.

Reply:
Fixed